### PR TITLE
Initial docs for Google Cloud

### PR DESCRIPTION
**Pull request by nickgros**
This feature is still behind alpha mode, so don't merge yet.

### CODE REVIEW

File: articles/custom_storage_location.md [hunk 1]
```diff
@@ -222,6 +222,117 @@ f <- synStore(f)
 
 Please see the [REST docs](http://docs.synapse.org/rest/org/sagebionetworks/repo/model/project/ExternalS3StorageLocationSetting.html){:target="_blank"} for more information on setting external storage location settings using our REST API.
 
+## Setting Up an External Google Cloud Storage Bucket
+
+Follow the documentation on Google Cloud's site to **[Create a Bucket](https://cloud.google.com/storage/docs/creating-buckets){:target="_blank"}**. 
+
```
Comment (xschildw) [inline articles/custom_storage_location.md]: The link and the button go to same page, should remove one?

Comment (nickgros) [reply articles/custom_storage_location.md]: Was just duplicating AWS instructions, but I removed it.

File: articles/custom_storage_location.md [hunk 2]
```diff
@@ -222,6 +222,115 @@ f <- synStore(f)
...
+##### Command line
+
+```bash
+# copy your owner.txt file to your s3 bucket
+gsutil cp owner.txt gs://nameofmybucket/nameofmyfolder
+```
+
+##### Web
+
+Navigate to your bucket on the Amazon Console and select the **Upload files** button to upload your text file.
```
Comment (brucehoff) [inline articles/custom_storage_location.md]: Should "Amazon Console" be "Google Cloud Platform Console"?

Comment (kdaily) [reply articles/custom_storage_location.md]:
```suggestion
Navigate to your bucket on the Google Cloud Console and select the **Upload files** button to upload your text file.
```

**Review by kdaily — CHANGES_REQUESTED** [review]

**Review by xschildw — APPROVED** [review]
OK

File: articles/custom_storage_location.md [hunk 3]
```diff
@@ -222,6 +222,115 @@ f <- synStore(f)
...
+
+project_destination = syn.restPOST('/projectSettings', body = json.dumps(project_destination))
+```
+
+##### R
+
+```r
+#set storage location
+library(synapser)
+synLogin()
```
Comment (xschildw) [inline articles/custom_storage_location.md]: missing import for toJSON below? Tried 'library(jsonlite).

**Review by xschildw — APPROVED** [review]
Add library() statement in R instructions for completeness.

**Review by xschildw — APPROVED** [review]

**Comment by kdaily** [comment]
I forgot to comment here that `master` is no longer the appropriate branch to merge during our effort to migrate to using GitHub pages. I will open an issue to translate these docs to the `gh-pages` branch until we are finished with the migration effort.

**Pull request merged**